c2corg_api/models/mailinglist.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy import Column, ForeignKey, String
+from sqlalchemy.orm import relationship
+
+from c2corg_api.models import Base, sympa_schema, users_schema
+from c2corg_api.models.user import User
+
+
+class Mailinglist(Base):
+    """
+    Class containing mailing list subscriptions.
+    Based upon Sympa mailing list server https://www.sympa.org/
+    """
+    __tablename__ = 'subscriber_table'
+    __table_args__ = {"schema": sympa_schema}
+
+    listname = Column('list_subscriber', String(50), nullable=False)
+    email = Column(
+        'user_subscriber', String(200),
+        ForeignKey(users_schema + '.user.email'), nullable=False)
```

Comment:
Making the email a foreign key might be problematic when the email is updated in the users table. Maybe it would make sense to have to columns `user_id` (foreign key on the users table) and `email`.

The `email` column could be updated with a trigger when the column `email` in the users table changes.